eureka/S3_data_reduction/miri.py

```diff
@@ -68,16 +68,36 @@ def read(filename, data, meta):
     # Record integration mid-times in BJD_TDB
     if len(int_times['int_mid_BJD_TDB']) == 0:
         if meta.firstFile:
-            print('  WARNING: The timestamps for the simulated MIRI data are currently hardcoded '
-                  'because they are not in the .fits files themselves')
+            print('  WARNING: The timestamps for the simulated MIRI data are currently hardcoded\n'+
+                  '           because they are not in the .fits files themselves')
         if 'WASP_80b' in data.attrs['filename'] and 'transit' in data.attrs['filename']:
             # Time array for WASP-80b MIRISIM transit observations
             # Assuming transit near August 1, 2022
-            time = np.linspace(2459791.3696221784, 2459791.6100991997, 4507, endpoint=True)[data.intstart - 1:data.intend-1]
+            phase_i = 0.95434
+            phase_f = 1.032726
+            t0 = 2456487.425006
+            per = 3.06785234
+            time_i = phase_i*per+t0
+            while np.abs(time_i-2459792.54237)>per:
+                time_i += per
+            time_f = phase_f*per+t0
+            while time_f<time_i:
+                time_f += per
+            time = np.linspace(time_i, time_f, 4507, endpoint=True)[data.attrs['intstart'] - 1:data.attrs['intend']-1]
         elif 'WASP_80b' in data.attrs['filename'] and 'eclipse' in data.attrs['filename']:
             # Time array for WASP-80b MIRISIM eclipse observations
             # Assuming eclipse near August 1, 2022
-            time = np.linspace(2459789.8356924183, 2459790.0761694396, 4506, endpoint=True)[data.intstart - 1:data.intend-1]
+            phase_i = 0.45434
+            phase_f = 0.532725929856498
+            t0 = 2456487.425006
+            per = 3.06785234
+            time_i = phase_i*per+t0
+            while np.abs(time_i-2459792.54237)>per:
+                time_i += per
+            time_f = phase_f*per+t0
+            while time_f<time_i:
+                time_f += per
+            data.time = np.linspace(time_i, time_f, 4506, endpoint=True)[data.attrs['intstart'] - 1:data.attrs['intend']-1]
```

Comment:
This line should also be just `time = ...`

Reply:
Nice catch!